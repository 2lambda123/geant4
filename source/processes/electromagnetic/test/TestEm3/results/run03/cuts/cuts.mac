# $Id: cuts.mac,v 1.1 1999-08-05 11:14:35 maire Exp $
#
# Macro file for "TestEm3.cc" 
# (can be run in batch, without graphic)
#
# Iron-Silicium 15 layers; electron 6, 8, 10, 12 GeV
#
# SICAPO Collaboration. CERN-PPE/91-218  Nov.91
#                       NIM A332 (85-90) Feb.93
#
/control/verbose 2
#
/calor/setNbOfLayers 15
/calor/setNbOfAbsor 2
/calor/setAbsor 0 Iron 23.08 mm
/calor/setAbsor 1 Silicium 0.4 mm
/calor/setSizeYZ 40. cm
/calor/update
/gun/setDefault
#
/run/particle/setCut 10. mm
/run/initialize
#
/run/verbose 2
/event/drawTracks none
/event/printModulo 10
/run/save on
#
/gun/particle e-
/gun/energy 6 GeV
/run/beamOn 500
