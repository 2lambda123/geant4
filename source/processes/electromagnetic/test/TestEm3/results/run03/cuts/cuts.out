

***** Table : Nb of materials = 16 *****

 Material: Aluminium     density:  2.700 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:   8.893 cm 
   --->  Element: Aluminium      Z = 13.0   N =  27.0   A =  26.98 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material: liquidArgon     density:  1.390 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:  14.065 cm 
   --->  Element: liquidArgon      Z = 18.0   N =  40.0   A =  39.95 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material:     Iron     density:  7.700 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:   1.797 cm 
   --->  Element:     Iron      Z = 26.0   N =  55.9   A =  55.85 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material:   Copper     density:  8.960 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:   1.436 cm 
   --->  Element:   Copper      Z = 29.0   N =  63.5   A =  63.55 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material: Tungsten     density: 19.300 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:   3.504 mm 
   --->  Element: Tungsten      Z = 74.0   N = 183.8   A = 183.85 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material:     Lead     density: 11.350 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:   5.612 mm 
   --->  Element:     Lead      Z = 82.0   N = 207.2   A = 207.19 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material:    Water     density:  1.000 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:  36.092 cm 
   --->  Element: Hydrogen  H   Z =  1.0   N =   1.0   A =   1.01 g/mole  fractionMass:  11.21 %  Abundance  66.67 %
   --->  Element:   Oxygen  O   Z =  8.0   N =  16.0   A =  16.00 g/mole  fractionMass:  88.79 %  Abundance  33.33 %

 Material: Scintillator     density:  1.032 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:  42.549 cm 
   --->  Element:   Carbon  C   Z =  6.0   N =  12.0   A =  12.01 g/mole  fractionMass:  91.45 %  Abundance  47.37 %
   --->  Element: Hydrogen  H   Z =  1.0   N =   1.0   A =   1.01 g/mole  fractionMass:   8.55 %  Abundance  52.63 %

 Material: Silicium     density:  2.330 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:   9.368 cm 
   --->  Element:  Silicon Si   Z = 14.0   N =  28.1   A =  28.09 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material:   quartz     density:  2.200 g/cm3   temperature: 273.15 K  pressure:   1.00 atm  RadLength:  12.295 cm 
   --->  Element:  Silicon Si   Z = 14.0   N =  28.1   A =  28.09 g/mole  fractionMass:  46.75 %  Abundance  33.33 %
   --->  Element:   Oxygen  O   Z =  8.0   N =  16.0   A =  16.00 g/mole  fractionMass:  53.25 %  Abundance  66.67 %

 Material:      Air     density:  1.290 mg/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength: 285.161 m  
   --->  Element: Nitrogen  N   Z =  7.0   N =  14.0   A =  14.01 g/mole  fractionMass:  70.00 %  Abundance  72.71 %
   --->  Element:   Oxygen  O   Z =  8.0   N =  16.0   A =  16.00 g/mole  fractionMass:  30.00 %  Abundance  27.29 %

 Material:  Aerogel     density: 200.000 mg/cm3  temperature: 273.15 K  pressure:   1.00 atm  RadLength:   1.493 m  
   --->  Element:  Silicon Si   Z = 14.0   N =  28.1   A =  28.09 g/mole  fractionMass:  29.22 %  Abundance  11.12 %
   --->  Element:   Oxygen  O   Z =  8.0   N =  16.0   A =  16.00 g/mole  fractionMass:  66.49 %  Abundance  44.42 %
   --->  Element: Hydrogen  H   Z =  1.0   N =   1.0   A =   1.01 g/mole  fractionMass:   4.19 %  Abundance  44.37 %
   --->  Element:   Carbon  C   Z =  6.0   N =  12.0   A =  12.01 g/mole  fractionMass:   0.10 %  Abundance   0.09 %

 Material: CarbonicGas     density: 27.000 mg/cm3  temperature: 325.00 K  pressure:  50.00 atm  RadLength:  13.406 m  
   --->  Element:   Carbon  C   Z =  6.0   N =  12.0   A =  12.01 g/mole  fractionMass:  27.29 %  Abundance  33.33 %
   --->  Element:   Oxygen  O   Z =  8.0   N =  16.0   A =  16.00 g/mole  fractionMass:  72.71 %  Abundance  66.67 %

 Material: WaterSteam     density:  0.300 mg/cm3  temperature: 500.00 K  pressure:   2.00 atm  RadLength:   1.203 km 
   --->  Element: Hydrogen  H   Z =  1.0   N =   1.0   A =   1.01 g/mole  fractionMass:  11.21 %  Abundance  66.67 %
   --->  Element:   Oxygen  O   Z =  8.0   N =  16.0   A =  16.00 g/mole  fractionMass:  88.79 %  Abundance  33.33 %

 Material: Galactic     density:  0.000 mg/cm3  temperature:   2.73 K  pressure:   0.00 atm  RadLength: 6317232936888469300000.000 km 
   --->  Element: Galactic      Z =  1.0   N =   1.0   A =   1.01 g/mole  fractionMass: 100.00 %  Abundance 100.00 %

 Material:     Beam     density:  0.010 mg/cm3  temperature: 273.15 K  pressure:   0.02 atm  RadLength:  36.786 km 
   --->  Element: Nitrogen  N   Z =  7.0   N =  14.0   A =  14.01 g/mole  fractionMass:  70.00 %  Abundance  72.71 %
   --->  Element:   Oxygen  O   Z =  8.0   N =  16.0   A =  16.00 g/mole  fractionMass:  30.00 %  Abundance  27.29 %



# Macro file for "TestEm3.cc" 
# (can be run in batch, without graphic)
#
# Iron-Silicium 15 layers; electron 6, 8, 10, 12 GeV
#
# SICAPO Collaboration. CERN-PPE/91-218  Nov.91
#                       NIM A332 (85-90) Feb.93
#
#
/calor/setNbOfLayers 15
/calor/setNbOfAbsor 2
/calor/setAbsor 0 Iron 23.08 mm
/calor/setAbsor 1 Silicium 0.4 mm
/calor/setSizeYZ 40. cm
/calor/update

-------------------------------------------------------------
 ---> The calorimeter is 15 layers of:
 	        Iron:  2.308 cm 
 	    Silicium:    400 mum
-------------------------------------------------------------
/gun/setDefault
#


/run/particle/setCut 0.1 mm
/run/initialize
Em3PhysicsList::SetCuts:CutLength : 100 mum

============= The cut Energy ==============================
                            gamma              e-             mu-          proton         neutron    
Cut in range                100 mum         100 mum         100 mum         100 mum         100 mum
Cut in energy
          Aluminium        2.31 keV         130 keV        1.38 MeV        3.39 MeV         990 eV 
        liquidArgon        1.99 keV        82.2 keV         869 keV        2.02 MeV         990 eV 
               Iron         6.2 keV         231 keV        2.27 MeV        5.61 MeV         990 eV 
             Copper        7.32 keV         249 keV        2.41 MeV        5.96 MeV         990 eV 
           Tungsten          36 keV         351 keV        3.11 MeV        7.35 MeV         990 eV 
               Lead        29.3 keV         240 keV         2.2 MeV        5.02 MeV         990 eV 
              Water         1.1 keV        84.3 keV         941 keV        2.28 MeV         990 eV 
       Scintillator         995 eV         85.3 keV         953 keV        2.34 MeV         990 eV 
           Silicium        2.29 keV         120 keV        1.28 MeV        3.14 MeV         990 eV 
             quartz        1.93 keV         120 keV        1.28 MeV        3.18 MeV         990 eV 
                Air         990 eV          990 eV         3.93 keV        1.62 keV         990 eV 
            Aerogel         990 eV         31.1 keV         335 keV         709 keV         990 eV 
        CarbonicGas         990 eV          990 eV         84.3 keV         179 keV         990 eV 
         WaterSteam         990 eV          990 eV          990 eV          990 eV          990 eV 
           Galactic         990 eV          990 eV          990 eV          990 eV          990 eV 
               Beam         990 eV          990 eV          990 eV          990 eV          990 eV 
===================================================
#
/run/verbose 2
/event/drawTracks none
/event/printModulo 10
/run/save on
#
/gun/particle e-
/gun/energy 6 GeV
/run/beamOn 500
### Run 0 start.
Start closing geometry.
Start Run processing.

Run terminated.
Run Summary
  Number of events processed : 500
  User=677.82s Real=841.07s Sys=145.52s

-------------------------------------------------------------
                                   total energy dep            total tracklen 
 
 Absorber0 (        Iron) :   5.93 GeV +- 38.51 MeV        5.23 m   +-  3.56 cm 
 Absorber1 (    Silicium) :  31.23 MeV +-  3.56 MeV        8.55 cm  +-  9.91 mm 

-------------------------------------------------------------




#
/run/particle/setCut 1. mm
/run/initialize
Em3PhysicsList::SetCuts:CutLength : 1 mm 

============= The cut Energy ==============================
                            gamma              e-             mu-          proton         neutron    
Cut in range                  1 mm            1 mm            1 mm            1 mm            1 mm 
Cut in energy
          Aluminium        6.89 keV         597 keV        5.02 MeV        13.1 MeV         990 eV 
        liquidArgon        6.18 keV         343 keV        3.24 MeV        8.32 MeV         990 eV 
               Iron        20.5 keV        1.25 MeV        8.42 MeV        21.6 MeV         990 eV 
             Copper        24.6 keV         1.4 MeV        8.95 MeV          23 MeV         990 eV 
           Tungsten         105 keV        2.31 MeV        11.7 MeV        29.5 MeV         990 eV 
               Lead         101 keV        1.38 MeV        8.52 MeV        20.9 MeV         990 eV 
              Water         2.9 keV         347 keV        3.39 MeV        8.95 MeV         990 eV 
       Scintillator        2.37 keV         356 keV        3.43 MeV        9.06 MeV         990 eV 
           Silicium        6.89 keV         541 keV        4.72 MeV        12.2 MeV         990 eV 
             quartz        5.52 keV         534 keV        4.66 MeV        12.2 MeV         990 eV 
                Air         990 eV          990 eV         48.4 keV        83.2 keV         990 eV 
            Aerogel        1.71 keV         119 keV        1.28 MeV        3.18 MeV         990 eV 
        CarbonicGas         990 eV         36.5 keV         393 keV         853 keV         990 eV 
         WaterSteam         990 eV          990 eV         11.9 keV        14.9 keV         990 eV 
           Galactic         990 eV          990 eV          990 eV          990 eV          990 eV 
               Beam         990 eV          990 eV          990 eV          990 eV          990 eV 
===================================================
#
/run/verbose 2
/event/drawTracks none
/event/printModulo 10
/run/save on
#
/gun/particle e-
/gun/energy 6 GeV
/run/beamOn 500
### Run 0 start.
Start closing geometry.
Start Run processing.

Run terminated.
Run Summary
  Number of events processed : 500
  User=552.15s Real=689.42s Sys=122.23s

-------------------------------------------------------------
                                   total energy dep            total tracklen 
 
 Absorber0 (        Iron) :   5.92 GeV +- 75.21 MeV        4.79 m   +-  6.47 cm 
 Absorber1 (    Silicium) :  30.84 MeV +-  3.70 MeV        8.33 cm  +-  1.05 cm 

-------------------------------------------------------------


#
/run/particle/setCut 10. mm
/run/initialize
Em3PhysicsList::SetCuts:CutLength : 1 cm 

============= The cut Energy ==============================
                            gamma              e-             mu-          proton         neutron    
Cut in range                  1 cm            1 cm            1 cm            1 cm            1 cm 
Cut in energy
          Aluminium        20.5 keV        4.55 MeV        18.5 MeV        47.8 MeV         990 eV 
        liquidArgon        19.1 keV        2.15 MeV        11.9 MeV        30.7 MeV         990 eV 
               Iron        60.1 keV        12.6 MeV        32.2 MeV        80.1 MeV         990 eV 
             Copper        70.2 keV        14.6 MeV        34.7 MeV        85.2 MeV         990 eV 
           Tungsten         358 keV        29.2 MeV        47.8 MeV         113 MeV         990 eV 
               Lead         282 keV          15 MeV        33.4 MeV        81.1 MeV         990 eV 
              Water        7.76 keV        2.15 MeV        12.2 MeV        32.2 MeV         990 eV 
       Scintillator         5.7 keV         2.2 MeV        12.3 MeV        32.6 MeV         990 eV 
           Silicium        20.7 keV        4.02 MeV        17.2 MeV        44.4 MeV         990 eV 
             quartz        16.1 keV        3.88 MeV        17.2 MeV        44.4 MeV         990 eV 
                Air         990 eV         8.07 keV         249 keV         521 keV         990 eV 
            Aerogel        4.78 keV         528 keV        4.63 MeV        12.2 MeV         990 eV 
        CarbonicGas        1.69 keV         140 keV         1.5 MeV        3.78 MeV         990 eV 
         WaterSteam         990 eV         1.17 keV         104 keV         223 keV         990 eV 
           Galactic         990 eV          990 eV          990 eV          990 eV          990 eV 
               Beam         990 eV          990 eV         2.78 keV           1 keV         990 eV 
===================================================
#
/run/verbose 2
/event/drawTracks none
/event/printModulo 10
/run/save on
#
/gun/particle e-
/gun/energy 6 GeV
/run/beamOn 500
### Run 0 start.
Start closing geometry.
Start Run processing.

Run terminated.
Run Summary
  Number of events processed : 500
  User=430.9s Real=541.08s Sys=102.72s

-------------------------------------------------------------
                                   total energy dep            total tracklen 
 
 Absorber0 (        Iron) :   5.93 GeV +- 44.06 MeV        3.83 m   +-  6.31 cm 
 Absorber1 (    Silicium) :  31.11 MeV +-  3.79 MeV        8.00 cm  +-  9.72 mm 

-------------------------------------------------------------
